apollo-client/src/main/java/com/ctrip/framework/apollo/util/ConfigUtil.java

```diff
@@ -375,6 +405,15 @@ public boolean isInLocalMode() {
     return false;
   }
 
+  public boolean isInKubernetesMode() {
```

Comment:
What's the use of this function?

Reply:
Indeed, It was not used. I will delete it